packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/host/dedicatedCloud-datacenter-host.controller.js

```diff
@@ -1,4 +1,5 @@
 import get from 'lodash/get';
+import set from 'lodash/set';
```

Comment:
we prefer not to use lodash any more

Reply:
Didn't know that, was keeping the same coding style as other files, removing it.